packages/nimble-components/src/menu-button/tests/menu-button.stories.ts

```diff
@@ -49,27 +47,72 @@ const metadata: Meta<MenuButtonArgs> = {
         }
     },
     argTypes: {
+        label: {
+            name: 'default',
+            description: 'The text content of the button.',
+            table: { category: 'slots' }
+        },
+        icon: {
+            name: 'start',
+            description:
+                'Set `slot="start"` to include an icon before the text content. This icon will be hidden when `content-hidden` is set.',
```

Comment:
> This icon will be hidden when `content-hidden` is set.

That isn't true. In fact, the `start` slot is the only thing that isn't hidden when `content-hidden` is set. Instead, this statement should probably be applied to the `default` slot of the button -- perhaps with a note that says that the default slot should still have content when `content-hidden` is set because it makes the button accessible with screen readers.

Reply:
Good catch, fixed by leveraging the shared doc strings from the pattern (and tweaking those strings)